### PR TITLE
doc: adding `Important` note for Azure OAuth Third-Party access requirements

### PR DESCRIPTION
**Pull request by ibuziuk**
## What does this pull request change?

**Before:**

image

**After:**

image

## What issues does this pull request fix or reference?

N/A

## Specify the version of the product this pull request applies to

**PR should be backported to 7.72.x for 3.8**

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Comment by ibuziuk** [comment]
@deerskindoll Vale borks on `via` but this is the exact wording that is used on Azure end
image

**Review by tolusha — APPROVED** [review]

**Review by deerskindoll — APPROVED** [review]
lgtm

File: modules/administration-guide/partials/proc_setting-up-the-microsoft-azure-devops-services-oauth-app.adoc [hunk 1]
```diff
@@ -49,3 +55,4 @@ $ echo -n '__<microsoft_azure_devops_services_oauth_client_secret>__' | base64 -
 .Additional resources
 
 * link:https://learn.microsoft.com/en-us/azure/devops/integrate/get-started/authentication/oauth?view=azure-devops[Authorize access to REST APIs with OAuth 2.0]
+* link:https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/change-application-access-policies?view=azure-devops[Change application connection & security policies for your organization]
```
Comment (max-cx) [inline modules/administration-guide/partials/proc_setting-up-the-microsoft-azure-devops-services-oauth-app.adoc:62]:
```suggestion
* link:https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/change-application-access-policies?view=azure-devops[Change application connection & security policies for your organization]

```

File: modules/administration-guide/partials/proc_setting-up-the-microsoft-azure-devops-services-oauth-app.adoc [hunk 2]
```diff
@@ -46,6 +54,10 @@ $ echo -n '__<microsoft_azure_devops_services_oauth_app_id>__' | base64
 $ echo -n '__<microsoft_azure_devops_services_oauth_client_secret>__' | base64 -w 0
 ----
 
+pass:[<!-- vale RedHat.TermsWarnings = YES -->]
+* link:https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/change-application-access-policies?view=azure-devops[Change application connection & security policies for your organization]
```
Comment (max-cx) [inline modules/administration-guide/partials/proc_setting-up-the-microsoft-azure-devops-services-oauth-app.adoc]:
```suggestion
```

**Pull request merged**